openedx/frontend-app-course-authoring
# feat: tag sections, subsections, and the whole course (FC-0053)

## Description
Adds the Manage tags menu item and TagCount button to Sections, Subsections and the whole Course.

![tag_course_outline](https://github.com/openedx/frontend-app-course-authoring/assets/849463/88e467e2-8d6c-47d5-91ab-f8b7aa0cc4e5)

## More Infomation
- Closes: https://github.com/openedx/modular-learning/issues/194

## Testing Instructions

- Enable contentstore.new_studio_mfe.use_new_course_outline_page waffle
- Run [Taxonomy Sample Data](https://github.com/open-craft/taxonomy-sample-data) script to create units with tags.
- Go to a course to display the course outline.
- Click in the `Manage Tags` in the course header and add some tags
- Verify the tag count component is updated in real time
- Use the `Manage tags` menu item in the three dots to add tags to Sections, Subsections and Units
- Verify the tag count component is updated in real time

___
Private ref: [FAL-3678](https://tasks.opencraft.com/browse/FAL-3678)
